docs/home/feature-management/features-and-variables/variables-and-variations.md

```diff
@@ -21,15 +21,15 @@ This will lead the user to a table containing all of the Variables used by this
 
 Each Feature contains its own set of Variables which it manages. **By default, upon creation of a Feature, a Boolean Variable will be created which has the same name as the Feature's key for easier reference.** Depending on the Feature type, the default Variations will be pre-set. The most common of which will be the Variations of "Variation OFF" and "Variation ON", with the boolean Variable being set to false and true, respectively.
 
-When a user is "Served" a Variation based on the [Targeting Rules](/home/feature-management/features-and-variables/targeting-users), the Variable Values the user receives on their devices or as an API response will be the values for the served Variation. 
+When a user is "Served" a Variation based on the [Targeting Rules](/home/feature-management/features-and-variables/targeting-users), the Variable Values the user receives on their devices or as an API response will be the values for the served Variation.
 
 A user can add as many Variables as they desire by simply clicking the "Add Variable" button. 
 
-![Variables Page with arrowing pointing at Add Variable button](/december_2021_variable-add.png)
+![Add Variable Modal](/april-2023-add-variable-modal.png)
 
-Give your new Variable a **name** as well as a **type**, as well as its **values** For each of the current Variations.
+Give your new Variable a **key** as well as a **type**, as well as its **values** For each of the current Variations.
```

Comment:
Give your new Variable a **key**, a **type**, and its **values** for each of the current Variations.